example/autograd/word_language_model/get_ptb_data.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+RNN_DIR=$(cd `dirname $0`; pwd)
+DATA_DIR="${RNN_DIR}/data/"
+
+if [[ ! -d "${DATA_DIR}" ]]; then
+  echo "${DATA_DIR} doesn't exist, will create one";
+  mkdir -p ${DATA_DIR}
+fi
+
+wget -P ${DATA_DIR} https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/ptb/ptb.train.txt;
```

Comment:
These can be moved to a dependency on mxnet.test_utils.download instead of a shell script.